less/control.less

```diff
@@ -158,10 +170,10 @@
 
   .Select-option {
     &.is-selected {
-      display: inline-block;
       background-color: initial;
-      padding: 0;
     }
+
+    padding: 8px 0 8px 22px;
```

Comment:
Please make this change only for the unique option

| Dev | Local |
| --- | --- |
| ![dev](https://user-images.githubusercontent.com/1847316/48511760-3ffc6000-e81d-11e8-9247-ea922213b63b.png) | ![local](https://user-images.githubusercontent.com/1847316/48511761-3ffc6000-e81d-11e8-9841-ef775eb17eb1.png) |

Reply:
same on the above, where you removed the `inline-block` and `padding`

| Dev | local |
| --- | --- |
| ![local](https://user-images.githubusercontent.com/1847316/48512292-c2d1ea80-e81e-11e8-9336-19f0f1f016b4.png) | ![dev](https://user-images.githubusercontent.com/1847316/48512293-c2d1ea80-e81e-11e8-8dee-8c721ecec166.png) |